Landing page

GregNicholas opened the pull request:
Global styles, css reset, theme colors were setup with styled components.
Please check Landing page code

AlphaNtihinduka commented on src/pages/Landing.jsx:
```diff
@@ -1,6 +1,124 @@
+  padding: 1rem;
+  text-align: center;
+
+  p {
```
- [ ] [OPTIONAL] You have done a great job using styled components 👏. However, to have maintainable features, I suggest considering creating different files for the styled-components and importing them into the Component file where it is required ☺️.
- [ ] [OPTIONAL] In addition, this will make our styled components reusable. For example, `getStartedButton` has the same styles as the `signup` and `login` buttons.

AlphaNtihinduka reviewed (changes requested):
## Hi @GregNicholas 🙋‍♂️
Good job working on your landing page. You have done the design to the best expectation and incredibly well. 👏  

You still need to implement a few on the project but you are almost there!!! 🚶: 

TO HIGHLIGHT:
- Correct use of styled components ✔️ 
- Your project is professional :heavy_check_mark: 
- Your repo is professional :heavy_check_mark: 
- Well-structured files :heavy_check_mark: 

# STATUS: Required Changes :recycle: 

![close](https://user-images.githubusercontent.com/99394374/210771148-619a4265-33b2-4e68-8901-6bb52693d6de.gif)

# Required Changes :recycle: 
 - Since I may not be able to access the `vite.config.js` file. I think I may use this to comment about the port. As CM suggested, please consider running the application on `port: 3000`. This will help us to easily connect to backend port: 3001 as suggested in the discord chat ☺️.
 You may consider changing the server to port: 3000 for the application to run on localhost: 3000
`server: {
    port: 3000, },` below plugins
 

## Optional suggestions
 - I strongly suggest considering using a professional Readme. Remember Readme file provides easy-to-find instructions for the project. This may include setup, installation, and run instructions including the license key.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or discord channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

GregNicholas commented:
I added server: { port: 3000, } in vite.config.js, added env file and put it in .gitignore, uninstalled bootstrap and styled components, and set up Tailwind. Then I styled the Landing.js page with Tailwind.

**Merge conflict. It looks like there's still one merge conflict. I tried to resolve it by deleting the part that was under the dev branch, but when I resolve, it wants me to click a button that says "Commit Merge." I didn't click it because I didn't want to merge myself without review. Thanks
